crates/codegen/schema/src/types/production.rs

```diff
@@ -92,7 +92,7 @@ impl<T> VersionMap<T> {
             VersionMap::Unversioned(t) => Some(t),
```

Comment:
To test/future-proof that change, should we implement/remove the `TODO` comment in `UnnamedFunctionDefinition` production in the grammar?

Testing this one will make sure we error out in the later versions as well.

Reply:
done

Reply:
Not sure I see the grammar change was pushed in the latest commit. Am I missing something?